src/overlays/actors/ovl_Dm_Zl/z_dm_zl.h

```diff
@@ -8,12 +8,25 @@ struct DmZl;
 typedef void (*DmZlActionFunc)(struct DmZl*, GlobalContext*);
 
 typedef struct DmZl {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x11C];
-    /* 0x0260 */ DmZlActionFunc actionFunc;
-    /* 0x0264 */ char unk_264[0x70];
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ UNK_TYPE1 pad188[0xD8]; // unused by DmZl
+    /* 0x260 */ DmZlActionFunc actionFunc;
+    /* 0x264 */ UNK_TYPE1 pad264[0x4C]; // unused by DmZl
```

Comment:
I bet these are the joint- and morphTables and a ColliderCylinder, based on what OoT has. I don't see any harm in typing them as such, even if they're unused.

Reply:
Not addressed yet?